components/DropdownLight/SubComponents/UseKeyPress.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from 'react';
+
+const useKeyPress = targetKey => {
+  const [keyPressed, setKeyPressed] = useState(false);
+
+  /* istanbul ignore next */
+  const downHandler = ({ key }) => {
+    if (key === targetKey) {
+      setKeyPressed(true);
+    }
+  };
+
+  /* istanbul ignore next */
+  const upHandler = ({ key }) => {
+    if (key === targetKey) {
+      setKeyPressed(false);
+    }
+  };
+
+  useEffect(() => {
+    window.addEventListener('keydown', downHandler);
+    window.addEventListener('keyup', upHandler);
+
+    return () => {
+      window.removeEventListener('keydown', downHandler);
+      window.removeEventListener('keyup', upHandler);
+    };
+  });
+
+  return keyPressed;
+};
+
+export default useKeyPress;
```

Comment:
I don't understand why this file was created. This is relative to another task, isn't it?

Reply:
that makes sense, I'll delete it